ProjectSidewalk/SidewalkWebpage
# Fixes neighborhood assignment infinite loop

As mentioned in issue #1158, there are some discrepancies between the `street_edge_assignment_count` table and the entries in the `audit_task` table. This caused a problem since we were calculating neighborhood completion percentage based on the entries in the `audit_task` table, but we were looking for new tasks to audit by looking at the `street_edge_assignment_count` table.

In fact, 76 entries were missing from the `street_edge_assignment_count` table, which led to the infinite loop. We knew that the region was not complete, yet we were not finding any unaudited streets to assign because of the missing values in that table.

To remedy the situation, I have removed all use of the `street_edge_assignment_count` table for now. I have also added the missing entries to that table via an evolutions file. So the table will continue to be updated as normal, we are just not using it right now, partially because there is at least one other bug in the table (see #1135).

Testing:
One way to test this would be to
1. Log in/sign up as a registered user
2. Find a region (like Monumental Core) which is stuck at ~90% complete
3. Running the develop branch, try auditing in that neighborhood, and verify that the completion percentage does not go up. (audit quickly by continually jumping, saying you aren't able to walk here)
4. Then check out the branch for this PR, and try auditing that neighborhood. Verify that the completion percentage _does_ go up.
5. Continue auditing until you get a region completion message.
6. Click to audit a new neighborhood, and verify that you do not get stuck in an infinite loop, like in the video posted on #1158 ;)
7. In the region you just finished, check the choropleth to see if it has reached 100%. You will probably have to truncate the `region_completion` table and reload the audit page, but it should say 100% at that point.

Resolves #1158
